my-app/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+import Card from './Card';
```

Comment:
Just a small thing, but since you're not rendering the `Card` here, you can go ahead and remove this import from this file. When you start your dev server, you'll see warnings if you have unused imports or variables, which means you can go ahead and delete.